### PR TITLE
Add admin activity to activity bar

### PR DESCRIPTION
**Pull request by guerler**
Adds an admin interface activity item to the activity bar.

image

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Comment by martenson** [comment]
lgtm, two minor ideas:

- Clicking on admin in masthead could open the activity bar directly, duplicating the activity bar admin button and providing consistency. Currently admin is shown the admin page together with tool panel which is not very useful.
- The "admin home" button would imho be better suited for "admin docs" linking to https://galaxyproject.org/admin/ and we could drop the "list of links" page. Later we could implement a proper dashboard in its place.

File: client/src/components/admin/AdminPanel.vue [hunk 1]
```diff
@@ -1,17 +1,135 @@
...
+import { useConfig } from "@/composables/config";
+
+import ActivityPanel from "@/components/Panels/ActivityPanel.vue";
+
+const { config, isConfigLoaded } = useConfig();
+
+const adminProperties = computed(() => {
+    return {
+        enableQuotas: config.value.enable_quotas,
+        isToolshedInstalled: config.value.tool_shed_urls && config.value.tool_shed_urls.length > 0,
```
Comment (martenson) [inline client/src/components/admin/AdminPanel.vue]:
```suggestion
        isToolshedInstalled: config.value.tool_shed_urls?.length > 0 ?? false,
```

**Comment by guerler** [comment]
@martenson thank you for looking into it. Clicking on the admin masthead tab now toggles the admin activity panel. Regarding linking the admin docs instead of admin home, I would prefer that we just go ahead and add a simple dashboard instead of linking to an external resource.

**Review by martenson — APPROVED** [review]

**Comment by martenson** [comment]
thanks @guerler !

**Pull request merged**